.gitmodules

```diff
@@ -31,3 +31,15 @@
 [submodule "thirdparty/ocl/clhpp_headers"]
 	path = thirdparty/ocl/clhpp_headers
 	url = https://github.com/KhronosGroup/OpenCL-CLHPP.git
+[submodule "thirdparty/onnx/onnx"]
+	path = thirdparty/onnx/onnx
+	url = https://github.com/openvinotoolkit/onnx.git
+[submodule "thirdparty/protobuf/protobuf"]
+	path = thirdparty/protobuf/protobuf
```

Comment:
maybe can we simplify to `thirdparty/protobuf`?

Reply:
`thirdparty/protobuf` contains a configuration CMakeLists.txt